src/passes/analysis/verilog.cpp

```diff
@@ -809,8 +843,8 @@ compile_module_body(RecordType *module_type,
     body.push_back(std::move(statement));
   }
   // Wire the outputs of the module and inout connections
-  assign_module_outputs(module_type, body, connection_map);
-  assign_inouts(definition->getSortedConnections(), body);
+  assign_module_outputs(module_type, body, connection_map, _inline);
+  assign_inouts(definition->getSortedConnections(), body, _inline);
```

Comment:
Should make these methods on the class so we can access the _inline member rather than having to pass them around.

Reply:
leave a TODO here for this